### PR TITLE
Fixed peer depencies, except some I can't alone?

### PR DESCRIPTION
**Pull request by filipesmedeiros**
Updated dependencies, but some I'm afraid to break. I just added carets and added peer depencies.

### CODE REVIEW

**Comment by filipesmedeiros** [comment]
Also, we should fix stuff that audit tells us about right? (Separate from this)

**Comment by alexandrebodin** [comment]
Hey @filipesmedeiros thanks for the help but you need to update the yarn.lock file before pushing those updates :)

**Comment by filipesmedeiros** [comment]
Hmm all tests passed but it failed anyway, not sure what happened

File: package.json [hunk 1]
```diff
@@ -2,15 +2,17 @@
   "private": true,
   "dependencies": {},
   "devDependencies": {
+    "@babel/core": "^7.0.0",
     "@babel/plugin-transform-modules-commonjs": "^7.9.0",
     "@testing-library/jest-dom": "^4.0.0",
     "@testing-library/react": "^9.1.0",
     "@testing-library/react-hooks": "^2.0.0",
+    "axios": "0.9.0",
```
Comment (soupette) [inline package.json:10]: Can you upgrade the version to `"axios": "^0.19.2",` so it matches the `strapi-admin` one?

File: package.json [hunk 2]
```diff
@@ -31,15 +33,21 @@
...
+    "knex": "0.20.0",
     "lerna": "^3.13.1",
     "lint-staged": "^9.2.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^1.18.2",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-is": "16.8.0",
     "react-test-renderer": "^16.9.0",
+    "redux-saga": "1.0.0",
```
Comment (soupette) [inline package.json:45]: Can you downgrade this version to `"redux-saga": "^0.16.0",` so it matches the strapi-admin and the strapi-plugin-users-permissions ones?

File: package.json (hunk 1 above)
Comment (filipesmedeiros) [reply package.json]: Sure thing!

**Comment by filipesmedeiros** [comment]
@alexandrebodin These changed messed up the linter?

**Comment by soupette** [comment]
> EDIT: I see that for some reason `strapi-plugin-content-manager` doesn't have buffet as a dependency, when it uses it (a bunch of times). Is this intended?


It is intended yes in fact the dependency is in the strapi-admin package currently. We will work on creating a common package that provides the necessary front-end dependencies.

**Comment by alexandrebodin** [comment]
Hi, I'm going to close this PR as it is outdated and was not updated recently. Feel free to discuss reopening it if you have time to finish it ;)

**Pull request closed without merging**